content/tracing/setup/dotnet.md

```diff
@@ -52,25 +52,92 @@ net start w3svc
 
 .NET Core doesn't support the Windows Global Assembly Cache (GAC) used by the .NET Framework. Instead, you can deploy Datadog's managed assemblies together with your application by adding the `Datadog.Trace.ClrProfiler.Managed` NuGet package. It is important that you match the versions of the NuGet package and the MSI installer.
 
-For example, if you are using the MSI installer for version `0.4.1-beta`, use:
+For example, if you are using the MSI installer for version `0.5.0-beta`, use:
 
 ```
-dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.4.1-beta
+dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.5.0-beta
 ```
 
 ### Linux
 
-Automatic instrumention for .NET Core on Linux is coming soon. To instrument your .NET Core applications on Linux today, use [manual instrumentation][8].
+Automatic instrumention is available on Linux for .NET Core 2.0+. To setup automatic instrumentation, 4 steps are necessary:
```

Comment:
- typo in "instrumention"
- "setup" (one word) is a noun or adjective, "to set up" is a verb

```suggestion
Automatic instrumentation is available on Linux for .NET Core 2.0+. To set up automatic instrumentation, 4 steps are necessary:
```